### PR TITLE
fix: blend shapes in audio buffer

### PR DESCRIPTION
**Pull request by HongSiu**
# Avatech SDK Hot Fix
- Updated @avatechai/avatars SDK to the latest version
- Change the SDK Audio setting to fit for audio buffer

# Summary
Change the audio initial setting with Avatar SDK, to fit for audio buffer. Also, enable blend shapes via audio init correctly.

# Before
https://github.com/Shaunwei/RealChar/assets/73209427/29c4774c-c4c3-498d-b5b9-91956c8b9f83

# After
https://github.com/Shaunwei/RealChar/assets/73209427/80acea2e-8714-49d1-8bce-0e40cacfb506

### CODE REVIEW

File: client/web/src/utils/audioUtils.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ const playAudio = (
     handleFirstInteractionAudio(); // For the first interaction, we need to play a sound to unlock blend shapes
     setInitialize(false);
   }
+  handleFirstInteractionAudio();
```
Comment (pycui) [inline client/web/src/utils/audioUtils.js:38]: this would be done twice for if (initialize) case. Is this expected?

Comment (HongSiu) [reply client/web/src/utils/audioUtils.js]: Oh thanks for reminding, missing out on the code

Comment (HongSiu) [reply client/web/src/utils/audioUtils.js]: Pushed changes

Comment (pycui) [reply client/web/src/utils/audioUtils.js]: Thanks, please request review again once this PR is ready

Comment (HongSiu) [reply client/web/src/utils/audioUtils.js]: ok, everything is good

**Pull request merged**